parachain/runtime/kusama-common/Cargo.toml

```diff
@@ -2,8 +2,9 @@
 name = "snowbridge-kusama-common"
 description = "Snowbridge Kusama Common"
 version = "0.0.1"
-authors = ["Snowfork <contact@snowfork.com>"]
-edition = "2021"
+edition.workspace = true
+authors.workspace = true
+repository.workspace = true
 license = "Apache-2.0"
```

Comment:
For completeness, can we add this field to all the manifests:

```toml
categories = ["cryptography::cryptocurrencies"]
```

Reply:
Done in https://github.com/Snowfork/snowbridge/pull/1101/commits/06b88e3b07d94e63b55f9ccf33a1d4e7b9adbbca.